### ISSUE TITLE
Is it possible for administrator to change password? Or to get password reset token for user?

### ISSUE THREAD

**Issue by louking** [Flask-Middleware/flask-security#474]
I have a user who, for whatever reason, isn't receiving the email for resetting password. I need to get this person online into my system. I don't see any way for me to change their password, e.g., `flask users create <user> --password TEXT` throws error.

I also don't see the url for password reset being logged. 

I know I can change the software and create an admin view to change the password, but would prefer not to deploy new release if possible.

**Comment by jwag956** [comment]
Hmm - Looking around - I don't think Flask-Security has that - though you aren't the first to request this - could easily be a new CLI command.

The only thing I can think to do is to go into the DB directly. You could do this by creating the password on your local dev machine (for any user), grab the contents of the DB and paste it into your production DB.

**Comment by louking** [comment]
Thanks for the quick response. For other people who see this, and my information, I believe for this to work the salt has to be the same on the development machine. Am I getting that right?

FWIW, I just discovered my server's IP address is being blocked by the apple infrastructure, or one of their vendors. I've logged a support ticket with proofpoint:

```
Apr 14 13:01:30 loutility-server-digitalocean postfix/smtp[23210]: 7BE394319A: to=<xxxxxxx@me.com>, relay=mx02.mail.icloud.com[17.42.251.12]:25, delay=71944, delays=71938/0.02/5.6/0, dsn=4.7.0, status=deferred (host mx02.mail.icloud.com[17.42.251.12] refused to talk to me: 550 5.7.0 Blocked - see https://support.proofpoint.com/dnsbl-lookup.cgi?ip=xxx.xxx.xxx.xxx)
```

**Comment by jwag956** [comment]
Yes - good point - SALT has to be the same.
